Source/Core/VideoBackends/CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
 add_subdirectory(OGL)
 add_subdirectory(Null)
 add_subdirectory(Software)
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
They should be changed to which, `WIN32` or `CMAKE_SYSTEM_NAME STREQUAL "Windows"`?